modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/MetaStorageManager.java

```diff
@@ -227,7 +227,7 @@ public void stop() throws Exception {
             IgniteUtils.closeAll(
```

Comment:
probably this should be called `runAll` then

Reply:
Why? It closes all `Closeables` given to it. Those are ot `Runnable`s.